mail/mailer.go

```diff
@@ -6,21 +6,42 @@
 package mail
 
 import (
+	"crypto/rand"
+	"fmt"
+	"io"
+	"math"
+	"math/big"
 	"net"
 	"net/smtp"
+	"strconv"
+	"strings"
+
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/jmhodges/clock"
 )
 
+type csprg interface {
```

Comment:
*csprg.

Note that as long as you are wrapping this you might as well wrap it at a higher level:

```
type idGenerator interface {
  generate() int64
}
```

And have a `realIDGenerator` that calls `rand.Int(reader, math.MaxInt64)` and panics on errors (since they will always be entropy failures).